R/arx_forecaster.R

```diff
@@ -2,7 +2,10 @@
 #'
 #' This is an autoregressive forecasting model for
 #' [epiprocess::epi_df] data. It does "direct" forecasting, meaning
-#' that it estimates a model for a particular target horizon.
+#' that it estimates a model for a particular target horizon. The
+#' forecaster trains one model using data over all available
+#' geo_value, and produces forecasts for `target_date = max(epi_df$time_value)
+#' + arx_arg_list()$ahead`.  
```

Comment:
This is almost the default, but not the only option. See [here](https://github.com/XuedaShen/epipredict/blob/26afaf58d7be5a29a55cdac0f080e39aa9347802/R/arx_forecaster.R#L134). And `ahead` is an argument that can be set inside `arx_arg_list()` as is `forecast_date` or `target_date`. Maybe better to point to that documentation, or an example?

Reply:
I see. Maybe we can something like by default predicts for `target_date = max(epi_df$time_value) + arx_arg_list()$ahead`.
And add an example that changes the `target_date`?